src/main/java/AutomationTrialQA2_superadminlogin/SuperAdminLogin.java

```diff
@@ -0,0 +1,92 @@
+package AutomationTrialQA2_superadminlogin;
```

Comment:
remove blank spaces
Remove commented code

Reply:
Done, I fixed this issue.